response-headers-usecases.md

```diff
@@ -0,0 +1,13 @@
+# Use Cases for response Headers
+
+The [Response Header Specification](https://github.com/w3c/trace-context/blob/master/spec/21-http_response_header_format.md) defines the format for response headers.
+
+As response headers solve (largely) different use cases, this doc aims to collect them and also address possible implications, like privacy issues.
+
+This document is a loose, unordered and ongoing collection.
+
+## Usecase 1: Report the proposed parent ID back to the caller 
+
```

Comment:
Here is another use case - Currently some of the Azure Control Plane APIs return Correlation Id as part of the response header which can then be used by the customer on reporting an issue or contacting us. We would like to have teams transition to using the standard W3C trace content and returning this as standard response header. Having the trace context in the form of the traceresponse header would make this easy for customers and enable us to pinpoint issues.